docs/fault-proof-alpha/immunefi.md

```diff
@@ -0,0 +1,86 @@
+    1. The `PreimageOracle` contract's local data storage must not be able to be corrupted by an external party.
+1. **op-program**
+    1. The [`op-program`][op-program] must produce a deterministic output given an identical setup state and input data.
+1. **Dispute Game Contracts**
+    1. Assuming the presence of an `honest challenger` (defined by the behavior of the [`op-challenger`][op-challenger]) participating within the game, the `FaultDisputeGame` utilizing the `MIPS` VM **must always** resolve favorably towards the honest L2 state.
+        1. *Note (1)*: The presence of an honest challenger implies that the honest challenger has exhausted all moves it would have made - any game where the honest challenger was unable to exhaust its move set can resolve unfavorably to their desired outcome. The aforementioned DoS vector is one such reason the honest challenger may not perform all its moves.
+1. **op-challenger**
+    1. The honest `op-challenger` must never make a claim that does not support the honest outcome of the dispute game (i.e., the outcome which favors the honest L2 state being considered canonical).
+        1. *Note:* Because of the rules in the current solving / resolution mechanism, the challenger will counter all claims on a level it disagrees with. While this is an inefficiency, it is not considered a violation of this invariant, as this behavior is necessary to ensure that all invalid claims have been countered.
+
+Any bug reports in the form of a PoC `op-e2e` test that demonstrates a violation of any of the above invariants will be considered valid bug reports and elligible for a reward.
+
+### Resources
+* [Cannon][cannon] & [Cannon Contracts][cannon-contracts]
+    * [Cannon VM Specs][cannon-vm-specs]
+* [`op-program`][op-program]
+    * [Fault Proof Specs][fault-proof-specs]
+* [Dispute Game][dispute-game]
+    * [Fault Dispute Game Specs][fault-dispute-specs]
+* [`op-challenger`][op-challenger]
+
+### Bounty Rewards
+All valid bug reports for Fault Proof Alpha qualify as a low-severity report. See [immunefi][immunefi]
+
+<!-- LINKS -->
+[cannon]: https://github.com/ethereum-optimism/optimism/tree/develop/cannon
+[cannon-vm-specs]: https://github.com/ethereum-optimism/optimism/blob/develop/specs/cannon-fault-proof-vm.md
+[dispute-game]: https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts-bedrock/src/dispute
+[fault-dispute-specs]: https://github.com/ethereum-optimism/optimism/blob/develop/specs/fault-dispute-game.md
+[cannon-contracts]: https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts-bedrock/src/cannon
+[op-program]: https://github.com/ethereum-optimism/optimism/tree/develop/op-program
+[op-challenger]: https://github.com/ethereum-optimism/optimism/tree/develop/op-challenger
+[alphabet-vm]: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/test/FaultDisputeGame.t.sol#L977-L1005
```

Comment:
nit: better to use a specific git revision (ex: https://github.com/ethereum-optimism/optimism/blob/c1cbacef0097c28f999e3655200e6bd0d4dba9f2/packages/contracts-bedrock/test/FaultDisputeGame.t.sol#L977) so the line numbers are stable.